metrics/README.md

```diff
@@ -0,0 +1,168 @@
+
+# Configure SPIRE Server and Agent to Emit Telemetry
+
+To demonstrate how to configure the SPIRE Server and the SPIRE Agent to emit telemetry we create a scenario using Docker Compose. We run a SPIRE deployment with two metric collectors, Statsd and Prometheus. Metrics collected by Statsd will be display using Graphite
```

Comment:
Yes, the SPIRE telemetry doc should be updated to have the correct form of StatsD. Unfortunately, the string to use in the config file is "Statsd =" so it's tricky.

Yes, good catch on the second to last sentence. Shows and demonstrates both work here, but since demonstrate was used earlier in the paragraph, let's go with shows. Yes, use metrics with a plural.

Reply:
Great, I'll apply those changes. Thanks.